### PR TITLE
[Reference] document new Doctrine APC cache service

### PR DESCRIPTION
**Pull request by xabbuh**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | yes (symfony/symfony#14429) |
| Applies to | 2.8+ |
| Fixed tickets |  |

### CODE REVIEW

**Comment by xabbuh** [comment]
Code PR was merged.

File: reference/configuration/framework.rst [hunk 1]
```diff
@@ -1372,6 +1372,14 @@ cache
 The service that is used to persist class metadata in a cache. The service
 has to implement the :class:`Symfony\\Component\\Validator\\Mapping\\Cache\\CacheInterface`.
 
+.. versionadded:: 2.8
+
```
Comment (WouterJ) [inline reference/configuration/framework.rst]: this empty line has to be removed

Comment (stof) [inline reference/configuration/framework.rst]: @WouterJ what is the progress on your bot checking such things automatically ? Having it running as a github hook on PRs would save time from reviewer by avoiding the need to report such mistakes manually

Comment (WouterJ) [inline reference/configuration/framework.rst]: I'm currently doing a lot of work making the current docs compatible with the bot.

As for the bot, I'm stumbling on more and more edge cases. At the moment, the bot is just reviewing plain text files, I have the feeling that I have to make it somewhat context aware (having the bot as custom sphinx builder/visitor isn't possible).
I'm also planning on limiting reports for the diff of the PR only.

Comment (xabbuh) [inline reference/configuration/framework.rst]: Thanks, good catch.

**Comment by weaverryan** [comment]
Sweet, thanks!

**Pull request merged**